dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
```

Comment:
- [ ]  Please pull request should have a descriptive title telling you what changes are inside.
- [ ] Please pull request should have a short summary describing introduced changes in more detail.
![image](https://user-images.githubusercontent.com/74744787/188256958-7ee0619c-034a-4cd3-b688-cfc86c3cc331.png)